zoho.go

```diff
@@ -0,0 +1,87 @@
+package zoholab
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/go-resty/resty/v2"
+	"github.com/pkg/errors"
+)
+
+const (
+	reportsUri   = "https://analyticsapi.zoho.eu"
+	AccountsUri  = "https://accounts.zoho.eu/oauth/v2/token"
+	validJson    = "true"
+	outputFormat = "JSON"
+	errorFormat  = "JSON"
+	apiVersion   = "1.0"
+	addRow       = "ADDROW"
+)
+
+// ZohoService is the struct for the zoho service.
+type ZohoService struct {
+	clientID     string
+	clientSecret string
+	refreshToken string
+	restyClient  *resty.Client
+}
+
+// NewZohoService instantiates a new zoho service.
+func NewZohoService(restyClient *resty.Client, clientID, clientSecret, refreshtoken string) *ZohoService {
+	return &ZohoService{
+		restyClient:  restyClient,
+		clientID:     clientID,
+		clientSecret: clientSecret,
+		refreshToken: refreshtoken,
+	}
+}
+
+// GetUri returns the URI for the specified table in the Zoho Analytics account.
+func (s *ZohoService) GetUri(emailID, dbName, tbName string) string {
+	// Join path is not used because the path must not be escaped.
+	return fmt.Sprintf("%s"+"/api/"+"%s"+"/"+"%s"+"/"+"%s", reportsUri, emailID, urlSplCharReplace(dbName), urlSplCharReplace(tbName))
+}
+
+// AddRow adds row to the specified table identified by the URI.
+func (s *ZohoService) AddRow(tableUri string, columnValues map[string]string) (interface{}, error) {
```

Comment:
Haben die rows die zurück kommen potenziell auch n model?

Reply:
Ja aber mit den wird nichts gemacht oder wollen wir nichts machen, ist das dann trotzdem notwendig?

Reply:
Nujoa wäre in so ner lib schon knorke